Client/index-react.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html class="no-js">
+  <head>
+    <meta charset="utf-8">
+    <title>Path Hero</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="style/style.css">
+  <!-- vendor scripts -->
+  <script src="bower_components/react/react.js"></script>
+  <script src="bower_components/react/JSXTransformer.js"></script>
+  <script src="app/app.js" type="text/jsx"></script>
+  <script src="https://code.jquery.com/jquery-1.11.2.js"></script>
```

Comment:
If any part of your app requires jquery this tag should be moved up in the order of execution.